cmake/libtinsConfig.cmake

```diff
@@ -1,16 +1,9 @@
 # - Config file for the libtins package
-# It defines the following variables
-#  LIBTINS_INCLUDE_DIRS - include directories for libtins
-#  LIBTINS_LIBRARIES    - libraries to link against
```

Comment:
Is there a way not to remove these? If people are currently relying on these being available, it's going to break their code.

Reply:
Honestly, I think this change will force all users to work with your library/CMake in correct way. 

What I tested so far I was not able produce the problem when it can break the build.
If I'm right It can break the build only in case when they are using the library in "header-only" mode. Is it possible to use the library in that mode?
If you know any problematic case feel free to share it with me.

Reply:
There's not really a "header only" mode. Going back to this change: we can't introduce a breaking change in the CMake script and just say "deal with it". Whatever code is out there that's using this file should still work after this change.

Reply:
I've just tried to convince you ;]  Hopefully I will find a time to update it this week.